kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -117,7 +117,6 @@ def kevm_prove(
                 'EVM.staticcall',
                 'EVM.create',
                 'EVM.create2',
-                'FOUNDRY.foundry.call',
```

Comment:
As `foundry_prove` calls `kevm_prove` internally, this small change would cause problems in `kontrol`. Before making this change, we should first refactor. I opened an issue: https://github.com/runtimeverification/evm-semantics/issues/2120

Reply:
Addressed: https://github.com/runtimeverification/evm-semantics/pull/2152